serverless.yml

```diff
@@ -148,6 +149,18 @@ functions:
           arn:
             Fn::GetAtt: [stackJanitorTable, StreamArn]
 
+  searchDeleteCloudformationStack:
+    handler: src/optional-handlers/searchDeleteCloudformationStack.handler
+    environment:
+      DEFAULT_EXPIRATION_PERIOD: ${self:custom.DEFAULT_EXPIRATION_PERIOD}
```

Comment:
Maybe have this as `${self:custom.DEFAULT_EXPIRATION_PERIOD} x 2` - not sure how math statements would resolve in yaml though